src/pages/components/palette.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Palette
+description: >
+  The palette provides the colors used within the Garden design system.
+package: '@zendeskgarden/react-theming'
+components:
+  - theming/src/elements/palette/index.ts
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To reference brand colors specific to Zendesk products</li>
+      <li>
+        <Markdown>To directly access `hue[shade]` color hex values</Markdown>
+      </li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To style custom components with Garden, access the palette through the
+          [`theme.palette`](/components/theme-object#palette) API or
+          [`withTheme`](/components/utilities#withTheme) utility instead
```

Comment:
Should we reference the `ThemeContext` object as that's the "new" approach provided by styled-components?

Reply:
Yeah, probably. The link to `withTheme` gets to that detail. But I was almost wondering if we wanted to ditch documenting `withTheme` on the website?

Reply:
To support class components I don't think the HOC is going anywhere on the styled-components side. So I guess that's a good reason to leave it around at least.

Reply:
I just elected to remove the `withTheme` clause here. In this context it is relevant to reference the theme object's key, less relevant to determine how to access (styled vs. non-styled Component) the key.